Scripts/script-ChangeRemediationSLAOnSevChange.yml

```diff
@@ -0,0 +1,45 @@
+fromversion: 4.1.0
+commonfields:
+  id: changeremediationslaonsevchange
+  version: -1
+name: ChangeRemediationSLAOnSevChange
+script: |
+  import datetime
+
+  # ##### Help #####
+  # Triggered Field Name is in: demisto.args()['name']
+  # Field's old value is in: demisto.args()['old']
+  # Field's new value is in: demisto.args()['new']
+  # Printing argument structure: demisto.results(demisto.args())
+
+  # ##### Configuration #####
+  CRITICAL_SLA = 60  # In minutes
+  NONCRITICAL_SLA = 6 # In days
+
+  # ##### Logic #####
+  args_sev =  demisto.args()['new']
+  now = datetime.datetime.utcnow()
+
+  if args_sev == 'Critical':
+      demisto.log('Severity changed to Critical')
```

Comment:
you demisto.log on purpose? To print the message to warroom?
because usually we don't leave demisto.log in the code

Reply:
Also, remember that this is an example script that they will have to copy and change according to their desire.